### PR TITLE
images: Add rhel-8-8 image

### PR DESCRIPTION
**Pull request by mvollmer**
* [x] image-refresh rhel-8-8

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-3830-20220906-094343/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-3830-20220906-094343/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-3830-20220906-095656/

**Comment by cockpituous** [comment]
image-refresh rhel-8-8 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-8-20220906-102312

**Comment by mvollmer** [comment]
Fixes to Cockpit in https://github.com/cockpit-project/cockpit/pull/17717, almost there...

File: lib/testmap.py [hunk 1]
```diff
@@ -53,8 +54,6 @@
             'fedora-testing',
             'fedora-testing/dnf-copr',
             'fedora-rawhide',
-            'rhel-8-8',
-            'rhel-9-2',
```
Comment (martinpitt) [inline lib/testmap.py:57]: That's gonna collide with #3828

**Comment by martinpitt** [comment]
@mvollmer LGTM, just needs that stupid little conflict. No need to run the tests again.

**Review by martinpitt — APPROVED** [review]
Resolved. Thanks!

**Pull request merged**